quiche/src/lib.rs

```diff
@@ -2950,10 +2950,6 @@ impl Connection {
 
         self.ack_eliciting_sent = false;
 
-        // Reset pacer and start a new burst when a valid
-        // packet is received.
-        self.paths.get_mut(recv_pid)?.recovery.pacer.reset(now);
```

Comment:
Sorry for commenting already merged comment.

This was added (AFAICT) because app calls quiche recv() and send() sequentially,
recv() may process ACK and update congestion control status, might lead to pacing rate change. 

Also, this change also impacts other CC with pacing (e.g. cubic + pacing). I think this need to be
a separate change from this PR and verify it has no negative impact.

Reply:
Yes, recv() and send() are called  sequentially and recv() may ultimately lead to pacing rate changes which is taken care in pacer methods. If resetting pacer is needed then its called from pacer methods when CC updates the pacer. We dont need to reset it here I think. This has been tested in various RTT environments.

Reply:
Since the pacer module assumes the current library behavior (https://github.com/cloudflare/quiche/blob/master/quiche/src/recovery/pacer.rs#L32-L34), the pacer status need to be cleared before we prepare a new series of send(). Without it, a small send().. -> recv() (let's say it's not ack) -> send().. may skew the pacer, because 2nd send() will continue from the previous pacer.used which is not an intended behavior. Not sure this was a concern for you.

Note that we already have some protection for [such case](https://github.com/cloudflare/quiche/blob/master/quiche/src/recovery/pacer.rs#L177), this may not be noticeable.

Reply:
pacer.used is always cleared when we hit the capacity or see packet with size less than previous packet size to match GSO requirements. Are you saying if we end up in a situation where cwnd allows to send just one packet and pacer.used is not cleared in that case ?

Reply:
Yes, e.g. send a full sized packet, recv, send a full size packet again. Hard to control from the app, but it could happen. (I've seen it)

Reply:
Ok got it, thanks for clarification.